lib/Sema/CSApply.cpp

```diff
@@ -8148,6 +8149,95 @@ bool ConstraintSystem::applySolutionFix(
                                       fix.first.getArgumentLabels(*this),
                                       isa<SubscriptExpr>(call->getFn()));
   }
+
+  case FixKind::AddConformace: {
```

Comment:
Can we extract this case into a different function? It's a ton of code

Reply:
I was thinking about doing that in the separate PR because I think we need to refactor `applySolutionFixes` into `Solution` itself, add `diagnoseFix`  instead of `applySolutionFix`, and extract this logic into `fixConformance`, WDYT?